### PR TITLE
added support for mysql time data type

### PR DESCRIPTION
**Pull request by b055**
This request is related to https://github.com/astaxie/beego/issues/1547

thanks

### CODE REVIEW

File: orm/db.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ import (
 )
 
 const (
+	formatTime    = "15:04:05"
```
Comment (astaxie) [inline orm/db.go]: it looks not format here

**Comment by astaxie** [comment]
--- FAIL: TestDataTypes (0.00s)
panic: reflect: call of reflect.Value.Uint on int64 Value [recovered]
	panic: reflect: call of reflect.Value.Uint on int64 Value

**Comment by b055** [comment]
How do you run these tests manually?

**Comment by b055** [comment]
```
685b358efb6b:beego b055$ go test orm/orm_test.go -v
## command-line-arguments
orm/orm_test.go:34: undefined: formatTime
orm/orm_test.go:35: undefined: formatDate
orm/orm_test.go:36: undefined: formatDateTime
orm/orm_test.go:65: undefined: ToStr
orm/orm_test.go:75: undefined: ToStr
orm/orm_test.go:171: undefined: GetDB
orm/orm_test.go:180: undefined: RegisterModel
orm/orm_test.go:180: undefined: Data
orm/orm_test.go:180: undefined: DataNull
orm/orm_test.go:180: too many errors
FAIL	command-line-arguments [build failed]
```

**Comment by astaxie** [comment]
```
go get -u github.com/go-sql-driver/mysql
go get -u github.com/mattn/go-sqlite3
go get -u github.com/lib/pq

#### MySQL
mysql -u root -e 'create database orm_test;'
export ORM_DRIVER=mysql
export ORM_SOURCE="root:@/orm_test?charset=utf8"
go test -v github.com/astaxie/beego/orm

#### Sqlite3
export ORM_DRIVER=sqlite3
export ORM_SOURCE='file:memory_test?mode=memory'
go test -v github.com/astaxie/beego/orm


#### PostgreSQL
psql -c 'create database orm_test;' -U postgres
export ORM_DRIVER=postgres
export ORM_SOURCE="user=postgres dbname=orm_test sslmode=disable"
go test -v github.com/astaxie/beego/orm
```

**Comment by astaxie** [comment]
@b055 As I have changed some code in ORM. please send the new PR again.

**Pull request closed without merging**